creator-node/src/logging.js

```diff
@@ -3,13 +3,32 @@ const shortid = require('shortid')
 
 const config = require('./config')
 
+// taken from: https://github.com/trentm/node-bunyan/issues/194#issuecomment-347801909
+// since there is no official support for string-based "level" values
+// response from author: https://github.com/trentm/node-bunyan/issues/194#issuecomment-70397668
+function RawStdOutWithLevelName() {
+  return {
+    write: (log) => {
+      // duplicate log object before sending to stdout
+      const clonedLog = Object.assign({}, log)
```

Comment:
seems like this is equivalent to `const clonedLog = log`?
i see you pulled this pattern from the above link, but that one does an additional assign in the same call

Reply:
nvm, i'm a dummy, this is a copy vs reference